csm/cli/support_bundle/support_bundle.py

```diff
@@ -189,14 +190,23 @@ async def bundle_status(command):
         :param command: Csm_cli Command Object :type: command
         :return: None
         """
-        bundle_id = command.options.get("bundle_id")
-        conf = GeneralConfig(Yaml(const.DATABASE_CLI_CONF).load())
-        db = DataBaseProvider(conf)
-        repo = SupportBundleRepository(db)
-        all_nodes_status = await repo.retrieve_all(bundle_id)
-        response = {"status": [each_status.to_primitive() for each_status in
+        try:
+            bundle_id = command.options.get("bundle_id", "")
+            conf = GeneralConfig(Yaml(const.DATABASE_CLI_CONF).load())
+            db = DataBaseProvider(conf)
+            repo = SupportBundleRepository(db)
+            all_nodes_status = await repo.retrieve_all(bundle_id)
+            response = {"status": [each_status.to_primitive() for each_status in
                                all_nodes_status]}
-        return Response(output = response, rc = CSM_OPERATION_SUCESSFUL)
+            return Response(output = response, rc = CSM_OPERATION_SUCESSFUL)
+        except DataAccessExternalError as e:
+            Log.error(f"Failed to connect to elasticsearch: {e}")
+            return Response(output = "Bundle status failed to connect to elasticsearch.",
+                            rc = str(errno.ECONNREFUSED))
+        except Exception as e:
+            Log.error(f"Failed to get bundle status: {e}")
+            return Response(output = "Failed to get status of bundle.",
```

Comment:
Please Change the Messages This is not an error we need to clarify the user that support bundle statues are not being saved as elasticsearch is down.
If we write Failed to get status it sounds as an error which is ideally expected here.

Reply:
We got this during fetching status. So we are not sure status got saved or not. Second exception block covers all cases.

Reply:
@kupranay @ajaysrivas please suggest message I should put for user. This is for main branch should I change in cortx also.

Reply:
Updated messages
Support Bundle status is not available currently as required services are down. Please wait and check the /tmp/support_bundle folder for newly generated support bundle.

Reply:
updated message